src/main/java/uk/gov/pay/directdebit/mandate/dao/MandateDao.java

```diff
@@ -55,6 +59,7 @@
             "  m.type AS mandate_type," +
             "  m.state AS mandate_state," +
             "  m.created_date AS mandate_created_date," +
+            "  m.creditor_id AS mandate_creditor_id," +
```

Comment:
As discussed with @DanailMinchev, we have the concept of `Mandate` which is purely internal to our system, and the concept of `GoCardlessMandate` which is the 'equivalent' mandate in gocardless. As `CreditorId` is 100% gocardless specific (it is an identifier used for one of their resources) - we should keep these models separate. `CreditorId` should be part of `GoCardlessMandate`, as, for example,`goCardlessMandateId`.

We can have a broader discussion on Monday if needed.

Reply:
Thank you, well spotted!
Created PR for database migrations - https://github.com/alphagov/pay-direct-debit-connector/pull/249
After #249 is merged we can address this comment.